packages/ember-runtime/lib/mixins/promise_proxy.js

```diff
@@ -40,7 +40,10 @@ function tap(proxy, promise) {
   A low level mixin making ObjectProxy promise-aware.
 
   ```javascript
-  let ObjectPromiseProxy = Ember.ObjectProxy.extend(Ember.PromiseProxyMixin);
+  import ObjectProxy from '@ember/object/proxy';
+  import PromiseProxyMixin from '@ember/object/promise-proxy-mixin';
+
+  let ObjectPromiseProxy = ObjectProxy.extend(PromiseProxyMixin);
```

Comment:
I'd probably use `const` here (since its in the module scope)...

Reply:
fixed.